File: front/src/app/core/services/seletores/Movimentacao.seletor.ts
import { Seletor } from "@app/core/services/seletores/SeletorBase";

export class MovimentacaoSeletor extends Seletor {
    Create(input: any) {
        throw new Error("Method not implemented.");
    }
    public descricao: string;
    public funcionarioCodigo: number;
   
}

File: back/ControleCustos.Api.Domain/Seletores/MovimentacaoSeletor.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ControleCustos.Api.Domain.Seletores
{
    public class MovimentacaoSeletor:SeletorBase
    {
		public int FuncionarioCodigo { get; set; }
		public string Descricao { get; set; }
	}
}
